trainee_template/derive_utilities.amlg

```diff
@@ -344,42 +344,61 @@
 						(let
 							(assoc
 								initial_values_list
-									;pull all unique id feature combos from dataset and sample n_samples worth
-									(map
-										(lambda
-											;a list of feature values for each unique conjuction
-											;each item in the list if there were two series id features would look like so:
-											;(list 12345 800)
-											(map
-												(lambda
-													;covnvert value back to number if feature is non-string
-													(if (contains_index numericNominalFeaturesMap (get series_id_features (current_index)))
-														(+ (current_value))
-														;else leave id as string
-														(current_value)
-													)
+									;if continuing an untrained series, use the id values from the specified data
+									(if (size continue_series_values)
```

Comment:
can we add continue_series_values/continue_series_features to the docstring for this method?